packages/compass-components/src/components/form-field-container.tsx

```diff
@@ -1,15 +1,16 @@
-import { spacing, css, cx } from '@mongodb-js/compass-components';
+import { spacing } from '@leafygreen-ui/tokens';
+import { css, cx } from '@leafygreen-ui/emotion';
 import React from 'react';
 
 const formFieldContainerStyles = css({
-  marginTop: spacing[4],
-  width: spacing[7] * 6,
```

Comment:
I couldn't find any reason for this to be hardcoded and it meant that we couldn't use FormFieldContainer for nested fields (ie. inside CollapsibleFieldContainer). If anyone can remember the history behind it and point at what's now broken please let me know and we can try and find a better way to do this.

Reply:
I understand why we may not want `width`, but shouldn't the margin be part of the component? the idea with the form field container is to align the spacing in every form

Reply:
ah we changed it to a vertical margin divided in half between top and bottom?